src/common/misc/VisItInit.C

```diff
@@ -109,6 +110,59 @@ NewHandler(void)
     abort(); // HOOKS_IGNORE
 }
 
+// ****************************************************************************
+//  Function: GetRelevantEnvVars()
+//
+//  Purpose:
+//      Creates a list of active env vars that are relevent to VisIt
+//      Used to write active env vars to logs
+//
+//  Programmer: Cyrus Harrison
+//  Creation:   August 21, 2001
+//
+//  Modifications:
+//
+// ****************************************************************************
+void
+GetRelevantEnvVars(std::vector<std::string> &res)
+{
+    res.clear();
+    
+    // check if the var starts with any of relevent pattern
+    std::vector<std::string> relevant_patterns = {"VISIT",
```

Comment:
It is possible there are some windows specific cases we should consider here, I didn't scan the visit.exe source.

Reply:
Some things from visit.exe: VISITHOME, VISITUSERHOME, PYTHONPATH, PYTHONHOME, LIBPATH, VISITSSH, VISITSSHARGS.  Not sure of the overlap of these with linux versions. Not all will be defined.

Reply:
Visit is crashing on startup on Windows. I will investigate further tomorrow.

Reply:
> Some things from visit.exe: VISITHOME, VISITUSERHOME, PYTHONPATH, PYTHONHOME, LIBPATH, VISITSSH, VISITSSHARGS. Not sure of the overlap of these with linux versions. Not all will be defined.

These should be captured, as the current logic looks for anything that starts with VISIT (VISITHOME, VISITUSERHOME, VISITSSH, VISITSSHARGS), anything that starts with PYTHON (PYTHONPATH, PYTHONHOME), and finally LIBPATH is a specific case it also looks for.

Reply:
Thanks, I think this is a good set to keep.